docs/user-guide/src/bmo/bmo.md

```diff
@@ -0,0 +1,42 @@
+<h1>Bare Metal Operator</h1>
+
+The Bare Metal Operator (BMO) is a custom Kubernetes controller that reconciles BareMetalHost resources using information provided by Ironic and the Cluster API Provider Metal3 (CAPM3) controller. A simplified information flow path is shown below:
+
+```bash
+CAPM3 ----------> BMO <----------> Ironic <----------> Server
+            BareMetalHost resource
+            - server MACAddress                        - image
+            - server address                           - cloud-init data
```

Comment:
I was wondering here if the `cloud-init data` should be here already or added after the explanation that it is coming from CAPM3? Maybe we could have instead things like CPU, disk and memory?

Reply:
I agree. 
Also the image should be ipa

Reply:
And cloud-init data does not have to exist at this stage.

Reply:
thanks, removed.